src/aiida/cmdline/commands/cmd_archive.py

```diff
@@ -210,9 +216,10 @@ def create(
     try:
         create_archive(entities, filename=output_file, archive_format=archive_format, **kwargs)
     except ArchiveExportError as exception:
-        echo.echo_critical(f'failed to write the archive file. Exception: {exception}')
+        echo.echo_critical(f'failed to write the archive file: {exception}')
     else:
-        echo.echo_success(f'wrote the export archive file to {output_file}')
+        if not dry_run:
+            echo.echo_success(f'wrote the export archive file to {output_file}')
```

Comment:
Do we perhaps still want a message here indicating the command worked successfully but this was a dry-run so no archive was actually created?

Reply:
I dunno, here's the current output

```
Report: Validating Nodes                                                                                                                   
Report: Test Run: Stopping before archive creation
Report: Archive would be created with:
----------------  -
users             1
computers         1
nodes             1
Repository Files  0
----------------  -
```

To me additional success green message for a test run seems a little strange.

Reply:
Ah, there are these message before that:
```
Report: Test Run: Stopping before archive creation
Report: Archive would be created with:
```
Fair enough, then it is not necessary indeed